### PR TITLE
submission flashrom: universal flash programming utility

### PR DESCRIPTION
**Pull request by ra1nb0w**
flashrom is a utility for identifying, reading, writing, verifying and
erasing flash chips.

The patch is needed because libusb 1.0.22 marked libusb_set_debug
as deprecated. With the next release could be deleted.

Tested with buspirate on
macOS 10.14.3 18D42
Xcode 10.1 10B61

###### Type(s)

- [x] submission
- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] checked your Portfile with `port lint`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: sysutils/flashrom/Portfile [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                flashrom
+version             1.0
+categories          sysutils
+platforms           darwin
+license             GPL-2
+maintainers         ra1nb0w openmaintainer
```
Comment (mf2k) [inline sysutils/flashrom/Portfile]: You need to prepend your Github handle with @. Can you also add your obfuscated email address? So the maintainers line will be like this:

`maintainers         {@ra1nb0w domain.com:name} openmaintainer`

File: sysutils/flashrom/Portfile [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                    size    321693
+
+# needed until next version
+patchfiles          patch-libusb_set_debug-deprecated.diff
+
+depends_build       port:pkgconfig
+
+depends_lib         port:libftdi0 \
+                    port:libusb-compat \
+                    port:libusb
```
Comment (mf2k) [inline sysutils/flashrom/Portfile]: Why are both of these libusb dependencies being added? It was my understanding that they are just different versions of the same library. So shouldn't libusb-compat be removed?

File: sysutils/flashrom/Portfile [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+use_configure       no
```
Comment (mf2k) [inline sysutils/flashrom/Portfile]: When using this option, you need to make sure you are [using the right compiler](https://trac.macports.org/wiki/UsingTheRightCompiler). This at least means either disabling the universal variant or making changes to ensure that it works.

File: sysutils/flashrom/Portfile (hunk 2 above)
Comment (ra1nb0w) [reply sysutils/flashrom/Portfile]: libusb-compat is a wrapper of libusb to provide compatibility with 0.1 version (used by flashrom). Now I see that libusb-compat has libusb as lib dependency. So I remove libusb. Thanks for checking.

File: sysutils/flashrom/Portfile (hunk 1 above)
Comment (ra1nb0w) [reply sysutils/flashrom/Portfile]: not too much obfuscated if always the same ;-)

File: sysutils/flashrom/Portfile (hunk 3 above)
Comment (ra1nb0w) [reply sysutils/flashrom/Portfile]: I didn't know this behaviour. thanks for spotting out

**Comment by ra1nb0w** [comment]
homebrew: SSL certificate problem: Invalid certificate chain.
How can i restart travis-ci?

**Comment by pmetzger** [comment]
Merged. Thanks, @ra1nb0w!

**Comment by ryandesign** [comment]
> * [x]  squashed and [minimized your commits](https://guide.macports.org/#project.github)?

Next time, please do actually squash your related commits.

**Comment by ra1nb0w** [comment]
Sure, I thought after merge ;-(

**Pull request merged**